webServer.js

```diff
@@ -248,10 +249,14 @@ function start() {
       }
       if (assets.gennedFileNames === undefined) {
         var fileArray = [];
+        var regexpr = "{*min.js,*min.css," + library.filename + "}";
```

Comment:
This is not human readable enough, I think I gave the example not the answer from the last review.

To improve a little bit:

```js
"{" + "*min.js, *min.css," + library.filename + "}";
```

Reply:
I think I requested to use `*.min.js` instead of `*min.js` from the last review already?

Reply:
@PeterDaveHello  Sorry, I forgot this.